examples/rbac/clusterrole/admin.yaml

```diff
@@ -0,0 +1,10 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: admin
```

Comment:
the name `admin` is used upstream for a bootstrap role intended for namespace-scoped users. `cluster-admin` is the equivalent superuser role.